DonjinKrawler-Server/src/main/java/donjinkrawler/Server.java

```diff
@@ -17,7 +19,7 @@
 public class Server {
     private static final com.esotericsoftware.kryonet.Server server =
             new com.esotericsoftware.kryonet.Server(16384 * 64, 16384 * 64);
-    private static final Map<Integer, PlayerData> playerConnectionMap = new HashMap<>();
+    public static final Map<Integer, PlayerData> playerConnectionMap = new HashMap<>();
```

Comment:
No need to have this field here if you're offloading the connections to a ConnectionManager class.

Reply:
:(